tutorials/create-esxi-cluster-dedibox/index.mdx

```diff
@@ -86,37 +60,37 @@ In this tutorial we are using VMware vSphere Hypervisor (ESXi) 6.7U3b. This vers
 
   <Lightbox src="scaleway-idrac_console.png" width="600" alt="" />
 
-7. The servers console displays. Click **Connect Virtual Media** in the menu on the top of it.
+7. Click **Connect Virtual Media** in the menu on the top of the virtual console window. A pop-up displays.
 
-8. A pop-up displays. Click **Choose file** (1) and select the downloaded VMware ESXi ISO file. Then click **Map device** (2) to connect the virtual CD-ROM drive to the server. Once done close the pop-up.
+8. Click **Choose file** (1) and select the downloaded VMware ESXi ISO file. Then click **Map device** (2) to connect the virtual CD-ROM drive to the server. Once done close the pop-up.
 
   <Lightbox src="scaleway-map_device.png" width="500" alt="" />
 
 9. Reboot your machine from the Dedibox console.
 
-10. During the boot process press **F11** on your keyboard to enter the boot manager:
+10. Press **F11** on your keyboard, during the boot process, to enter the boot manager:
 
   <Lightbox src="scaleway-boot_manager.png" width="400" alt="" />
 
-11. Once the boot manager displays, click **One-shot BIOS Menu**:
+11. Click **One-shot BIOS Menu** in the boot manager:
 
   <Lightbox src="scaleway-one_shoot_boot.png" width="500" alt="" />
 
-12. Click the **Virtual Optical Drive** to boot the machine from the connected ISO file:
+12. Select the **Virtual Optical Drive** to boot the machine from the connected ISO file:
 
   <Lightbox src="scaleway-virtual_drive.png" width="500" alt="" />
 
-13. The server is booting from the ISO file. Depending on your Internet connection, this may take a while.
+13. The server is booting from the ISO file. Depending on your Internet connection, this may take a while. Once ready, the ESXi installer displays. 
```

Comment:
I think this shouldn't be a numbered step, since there is no action.

Reply:
```suggestion
The server is booting from the ISO file. Depending on your Internet connection, this may take a while. Once ready, the ESXi installer displays. 
```